types/serialization.go

```diff
@@ -246,6 +246,8 @@ func (s *State) FromProto(other *pb.State) error {
 		return err
 	}
 	s.LastHeightValidatorsChanged = other.LastHeightValidatorsChanged
+	s.ConsensusParams = other.ConsensusParams
+	s.LastHeightConsensusParamsChanged = s.LastHeightConsensusParamsChanged
```

Comment:
Should this be assigning from `other.LastHeightConsensusParamsChanged`?

Reply:
Yes, already fixed.